src/Files.App/Views/SettingsPages/Tags.xaml

```diff
@@ -221,8 +221,34 @@
 												Padding="8,4"
 												Background="Transparent"
 												BorderBrush="Transparent"
-												Click="RemoveTag_Click"
-												Content="{helpers:ResourceString Name=Delete}" />
+												Click="PreRemoveTag_Click"
+												Content="{helpers:ResourceString Name=Delete}">
+												<Button.Flyout>
+													<Flyout>
+														<StackPanel
+															MaxWidth="150"
+															Padding="0"
+															Spacing="12">
+															<TextBlock
+																Text="{helpers:ResourceString Name=ConfirmDeleteTag}"
+																TextAlignment="Center"
+																TextWrapping="Wrap" />
+															<StackPanel
+																HorizontalAlignment="Center"
+																Orientation="Horizontal"
+																Spacing="12">
```

Comment:
Can you try with 8? The ones in the list are spaced further apart because there were multiple items in the row and it was too crowded.